backend.md

```diff
@@ -1,75 +1,74 @@
 # Backend Challenge
+> *Este arquivo também está disponível em português [aqui](backend_pt.md).*
 
-## About the Challenge
+In Zé we thrive to find our best partner to deliver beverages to our customers, providing the best and fastest service.
+To achieve this our compute fleet deals with GIS objects all the time.
 
-In this [json](files/pdvs.json) you will find data about hundreds of partners with the following format:
-```javascript
+When programming we try to follow a myriad of coding best practices and patterns (those you can read on books like Clean Code, Clean Architecture, The Pragmatic Programmer, Domain-Driven Design, Microservice Patterns, etc...). 
+Since writing **good code is a must** on our daily basis we expect people that want to join our team think alike. This code challenge is something we designed to find this awesome programmer.
```

Comment:
```suggestion
Since writing **good code is a must** on our daily basis we expect people that want to join our team to think alike. This code challenge is something we designed to find this awesome programmer.
```